composer.json

```diff
@@ -14,11 +14,12 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.4 || ^8.0",
```

Comment:
is the version drop required ? could we drop less of them ?

Reply:
Why would you like to support end-of-life versions of PHP? I think for people using old versions of PHP the older library versions will work. 

Tools using this library can easily run on newer versions while the applications being processed can be on old PHP versions.

Reply:
Because that would mean that I also need to drop PHP versions. I am not saying this is bad but if it can be avoided because x.y is supported by the lib you add then let's not drop them ?